db/migrate/20150303140556_add_transaction_process_id_to_transaction_type.rb

```diff
@@ -0,0 +1,10 @@
+class AddTransactionProcessIdToTransactionType < ActiveRecord::Migration
+  def up
+    add_column :transaction_types, :transaction_process_id, :int, after: :community_id
+    add_index :transaction_types, :transaction_process_id
```

Comment:
What do we need this index for? What's the use case for looking up a transaction type by transaction process?